deployments/charts/kuma/templates/_helpers.tpl

```diff
@@ -273,6 +274,10 @@ env:
 - name: KUMA_EXPERIMENTAL_GATEWAY_API
   value: "true"
 {{- end }}
+{{- if .Values.experimental.cni }}
+- name: KUMA_EXPERIMENTAL_CNI
+  value: "true"
```

Comment:
If this was cni app label and take the value of `app` (default would be ""). Wouldn't that remove the need for `parent.app` ?